doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1,4401 +1,15 @@
-
-Secure Partition Manager (SPM)
-==============================
-
-.. note::
-    The Secure Partition Manager (SPM) is deprecated as of |NCS| v2.1.0 and removed after |NCS| v2.2.0. It is replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
-
-.. rst-class:: v2-2-0
-
-NCSDK-19156: Building SPM for other boards than ``nrf5340dk_nrf5340_cpuapp`` and ``nrf9160dk_nrf9160`` fails with compilation error in :file:`cortex_m_systick.c`
-  This happens because the :kconfig:option:`CONFIG_CORTEX_M_SYSTICK` configuration option is enabled while the systick node is disabled in the devicetree.
-
-  **Affected platforms:** nRF9160, nRF5340
-
-  **Workaround:** Enable the systick node in a DTS overlay file for the SPM build by completing the following steps:
-
-  1. Create an overlay file :file:`systick_enabled.overlay` with the following content:
-
-     .. code-block::
-
-        &systick {
-          status = "okay";
-        };
-
-  #. Add the overlay file as a build argument to SPM:
-
-     .. parsed-literal::
-       :class: highlight
-
-       west build -- -Dspm_DTC_OVERLAY_FILE=systick_enabled.overlay
-
-.. rst-class:: v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
-
-NCSIDB-114: Default logging causes crash
-  Enabling default logging in the Secure Partition Manager sample makes it crash if the sample logs any data after the application has booted (for example, during a SecureFault, or in a secure service).
-  At that point, RTC1 and UARTE0 are non-secure.
-
-  **Workaround:** Do not enable logging and add a breakpoint in the fault handling, or try a different logging backend.
-
-.. rst-class:: v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0
-
-NCSDK-8232: Secure Partition Manager and application building together
-  It is not possible to build and program Secure Partition Manager and the application individually.
-
-.. rst-class:: v1-5-2 v1-5-1 v1-5-0
-
-CIA-248: Samples with default SPM config fails to build for ``thingy91_nrf9160_ns``
-   All samples using the default SPM config fails to build for the ``thingy91_nrf9160_ns``  build target if the sample is not set up with MCUboot.
-
-   **Affected platforms:** Thingy:91
-
-   **Workaround:** Use the main branch.
-
-Zephyr repository issues
-************************
-
-In addition to these known issues, check the current issues in the `official Zephyr repository`_, since these might apply to the |NCS| fork of the Zephyr repository as well.
-
-Reporting |NCS| issues
-**********************
-
-To get help and report issues that are not related to Zephyr but to the |NCS|, go to Nordic's `DevZone`_.
+Known issues are only tracked for the latest official release.
```

Comment:
Isn't this release the latest official release? Also, the known issues are tracked for all releases, not only for the latest official one. 

I think this sentence is not needed and can be removed. Having only the "See ..." sentence is enough.